File: app/src/main/java/uk/co/agfsoft/skyweather/view/FavouritesView.java
package uk.co.agfsoft.skyweather.view;

import java.util.List;

import uk.co.agfsoft.skyweather.model.WeatherCity;



public interface FavouritesView {
    void showAddFavouriteView();

    void showIntroOverlay(boolean showView);

    void updateFavourites(List<WeatherCity> favourites);

    void showWeatherForCity(WeatherCity weatherCity);
}

File: app/src/main/kotlin/uk/co/agfsoft/skyweather/view/FavouritesView.kt
package uk.co.agfsoft.skyweather.view

import model.WeatherCity


interface FavouritesView {
    fun showAddFavouriteView()

    fun showIntroOverlay(showView: Boolean)

    fun updateFavourites(favourites: List<WeatherCity>)

    fun showWeatherForCity(weatherCity: WeatherCity)
}
